smt/expr.cpp

```diff
@@ -675,6 +696,17 @@ expr expr::ctpop() const {
   return res;
 }
 
+// TODO: make rounding mode customizable
```

Comment:
not sure we care about customizing rouding mode?

Reply:
I think we should care about supporting non-default RMs, I noticed that LLVM now has a new set of experiment intrinsics called "constrained floating points" which allows user to specify RM by their own, check https://llvm.org/docs/LangRef.html#constrained-floating-point-intrinsics . Also the lifejacket paper also list supporting various RMs as a limitation to their work.